frog/imports/client/Wiki/index.js

```diff
@@ -520,6 +595,82 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     return { liId, pageId };
   };
 
+  editAccess = async action => {
```

Comment:
Break this function into smaller parts that are responsible for editing access to different features

Reply:
Why?

Reply:
Functions should do one thing, and only one thing